### PR TITLE
theme: Add Everblush

### PR DESCRIPTION
**Pull request by IsotoxalDev**
### [Everblush](https://everblush.github.io/)
Everblush is an Independent, Dark, Vibrant and Beautiful Colorscheme for helix !!

image

### CODE REVIEW

**Comment by philipgiuliani** [comment]
The whitespace characters seem to be way to strong:
## Helix
helix

## VSCode
vscode

**Comment by IsotoxalDev** [comment]
I don't usually have whitespaces, didn't notice

File: runtime/themes/everblush.toml [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+"ui.selection.primary" = { bg = "black" }
+"ui.statusline" = { fg = "foreground", bg = "backound" }
+"ui.statusline.inactive" = { fg = "foreground", bg = "backgoround" }
+"ui.statusline.normal" = { fg = "white", bg = "background" }
+"ui.statusline.insert" = { fg = "blue", bg = "background" }
+"ui.statusline.select" = { fg = "cyan", bg = "magenta" }
+"ui.text" = { fg = "foreground" }
+"ui.text.focus" = { fg = "blue" }
+"ui.virtual.ruler" = { bg = "comment" }
+"ui.virtual.whitespace" = { fg = "comment" }
```
Comment (the-mikedavis) [inline runtime/themes/everblush.toml:88]: There are some other `ui.virtual` scopes that should be styled for features like soft-wrap and indentation guides: https://docs.helix-editor.com/master/themes.html#interface

You might be able to re-use the `comment` highlight for everything but the rulers depending on how that looks to you:

```toml
"ui.virtual.ruler" = { bg = "comment" }
"ui.virtual" = { fg = "comment" }
```

Comment (IsotoxalDev) [reply runtime/themes/everblush.toml]: Thanks for pointing out. I have implemented warp and indent guide. Also set the ruler to a bit different color as it maybe confusing along with other virtual contents that may be in the near the ruler. Thanks!!

File: runtime/themes/everblush.toml [hunk 2]
```diff
@@ -0,0 +1,132 @@
+# Author: Isotoxal <isotoxal@proton.me>
+
+"attribute" = { fg = "blue" }
+"boolean" = { fg = "cyan" }
+"character" = { fg = "blue" }
```
Comment (the-mikedavis) [inline runtime/themes/everblush.toml:5]: Some of these scopes aren't used by Helix. These should be

```toml
"constant.builtin.boolean" = { fg = "cyan" }
"constant.character" = { fg = "blue" }
```

The captures in use are documented here: https://docs.helix-editor.com/master/themes.html#syntax-highlighting

File: runtime/themes/everblush.toml [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+"text.environtment" = { fg = "magenta" }
+"text.literal" = { fg = "green", italic = true }
+"text.math" = { fg = "cyan" }
+"text.note" = { fg = "black-light" }
+"text.reference" = { fg = "cyan" }
+"text.strike" = { fg = "white", strikethrough = true }
+"text.strong" = { fg = "white", bold = true }
+"text.title" = { fg = "yellow", bold = true }
+"text.underline" = { fg = "magenta", underline = true }
+"text.uri" = { fg = "yellow", underline = true }
```
Comment (the-mikedavis) [inline runtime/themes/everblush.toml:56]: Modifiers like strikethrough/bold/underline/italic should be specified in the `modifiers` list instead (https://docs.helix-editor.com/master/themes.html#modifiers):

```toml
"text.strike" = { fg = "white", modifiers = ["strikethrough"] }
```

These should be `markup.*` scopes instead though, for example https://github.com/helix-editor/helix/blob/98a3d46912be7dcc650c54ea417d7f00ab6d05a3/runtime/themes/darcula.toml#L56-L63

File: runtime/themes/everblush.toml (hunk 2 above)
Comment (IsotoxalDev) [reply runtime/themes/everblush.toml:5]: I am really sorry!!, I copied out a neovim theme's tree sitter and run a macro on it to convert it to toml. I didn't realize helix may not have some of the things used. Thank you!!!

File: runtime/themes/everblush.toml [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+
+"error" = { fg = "red" }
+"hint" = { fg = "green" }
+"warning" = { fg = "yellow" }
+"info" = { fg = "blue" }
+"diagnostic.error" = { underline = { style = "curl", color = "red" } }
+"diagnostic.warning" = { underline = { style = "curl", color = "yellow" } }
+"diagnostic.info" = { underline = { style = "curl", color = "blue" } }
+"diagnostic.hint" = { underline = { style = "curl", color = "green" } }
+"special" = { fg = "orange" }
```
Comment (the-mikedavis) [inline runtime/themes/everblush.toml:81]: It looks like `orange` isn't defined in the palette

File: runtime/themes/everblush.toml [hunk 5]
```diff
@@ -0,0 +1,114 @@
...
+"ui.help" = { fg = "foreground", bg = "contrast" }
+"ui.linenr" = { fg = "comment" }
+"ui.linenr.selected" = { fg = "foreground" }
+"ui.menu" = { fg = "foreground", bg = "contrast" }
+"ui.menu.selected" = { bg = "black" }
+"ui.popup" = { fg = "foreground", bg = "contrast" }
+"ui.selection" = { bg = "black" }
+"ui.selection.primary" = { bg = "black" }
+"ui.statusline" = { fg = "foreground", bg = "backound" }
+"ui.statusline.inactive" = { fg = "foreground", bg = "backgoround" }
```
Comment (the-mikedavis) [inline runtime/themes/everblush.toml:61]:
```suggestion
"ui.statusline.inactive" = { fg = "foreground", bg = "background" }
```

File: runtime/themes/everblush.toml [hunk 6]
```diff
@@ -0,0 +1,114 @@
...
+"ui.cursorline.primary" = { bg = "cursorline" }
+"ui.help" = { fg = "foreground", bg = "contrast" }
+"ui.linenr" = { fg = "comment" }
+"ui.linenr.selected" = { fg = "foreground" }
+"ui.menu" = { fg = "foreground", bg = "contrast" }
+"ui.menu.selected" = { bg = "black" }
+"ui.popup" = { fg = "foreground", bg = "contrast" }
+"ui.selection" = { bg = "black" }
+"ui.selection.primary" = { bg = "black" }
+"ui.statusline" = { fg = "foreground", bg = "backound" }
```
Comment (the-mikedavis) [inline runtime/themes/everblush.toml:60]:
```suggestion
"ui.statusline" = { fg = "foreground", bg = "background" }
```

File: runtime/themes/everblush.toml [hunk 7]
```diff
@@ -0,0 +1,114 @@
+# Author: Isotoxal <isotoxal@proton.me>
+
+"attribute" = { fg = "blue" }
+"comment" = { fg = "comment", italic = true }
```
Comment (the-mikedavis) [inline runtime/themes/everblush.toml:4]:
```suggestion
"comment" = { fg = "comment", modifiers = ["italic"] }
```

File: runtime/themes/everblush.toml (hunk 4 above)
Comment (IsotoxalDev) [reply runtime/themes/everblush.toml]: ah oops

**Review by the-mikedavis — APPROVED** [review]

**Pull request merged**